src/ConfigurationChief.ts

```diff
@@ -51,6 +51,7 @@ const defaultConfig: Configuration = {
   ignore: [],
   ignoreBinaries: [],
   ignoreDependencies: [],
+  ignoreExportsUsedInFile: false,
```

Comment:
This feels awfully specific for a high-level option, but I couldn't think of a better way to position it... 😕